azure-pipelines.yml

```diff
@@ -23,6 +23,17 @@ jobs:
     APPVEYOR_REPO_BRANCH: $[coalesce(variables['System.PullRequest.TargetBranch'], variables['Build.SourceBranchName'])]
     APPVEYOR_REPO_COMMIT: $(Build.SourceVersion)
   steps:
+  - pwsh: |
+      $isReleaseBuild = $false
+      if ($env:BuildSourceBranchName -like "release_4.0-*")
```

Comment:
Should this `"release_4.0-*"` be a pipeline variable? This way we can change the target branch for a release without having to make a code change.